frontend/Caddyfile

```diff
@@ -0,0 +1,22 @@
+# SPDX-FileCopyrightText: Copyright DB InfraGO AG and contributors
+# SPDX-License-Identifier: Apache-2.0
+
+:8080 {
+        log {
+                output file /var/log/access.log
+        }
+
+        root * /srv
+
+        # Serve static assets with aggressive caching
+        header /assets/* Cache-Control "public, max-age=31536000, immutable"
+
+        try_files {path} /index.html
+        # Force browsers to always load index.html
+        header /index.html Cache-Control "public, max-age=0, must-revalidate"
```

Comment:
Isn't this the default?

Reply:
Caddy's [guide for single-page apps](https://caddyserver.com/docs/caddyfile/patterns#single-page-apps-spas) says to do this when using hashed filenames